includes/class-sensei-learner.php

```diff
@@ -218,11 +223,21 @@ public function get_enrolled_courses_query_args( $user_id, $base_query_args = []
 			'include_children' => false,
 		];
 
-		return $query_args;
+		/**
+		 * Filters the arguments of the query which fetches a user's enrolled courses.
+		 *
+		 * @since 3.3.0
+		 *
+		 * @param array $query_args  The query args.
+		 * @param int   $user_id     The user id.
+		 */
+		return apply_filters( 'sensei_user_enrolled_courses_args', $query_args, $user_id );
```

Comment:
Now that we have [hook docs](https://automattic.github.io/sensei/), we'll need to switch to using the JSDoc format. You can see an example [here](https://github.com/Automattic/sensei/pull/3067/commits/5b04b8baceeb3f3c1bfbe030502b9231ee13afa3).

Reply:
Fixed!